src/ui/common/src/utils/SupportedIntegrations.tsx

```diff
@@ -0,0 +1,465 @@
+import { AirflowDialog, getAirflowValidationSchema } from "../components/integrations/dialogs/airflowDialog";
+import { AthenaDialog, getAthenaValidationSchema } from "../components/integrations/dialogs/athenaDialog";
+import { AWSDialog, getAWSValidationSchema } from "../components/integrations/dialogs/awsDialog";
+import AzureDialog, { getAzureValidationSchema } from "../components/integrations/dialogs/azureDialog";
+import { BigQueryDialog, getBigQueryValidationSchema } from "../components/integrations/dialogs/bigqueryDialog";
+import { CondaDialog } from "../components/integrations/dialogs/condaDialog";
+import { DatabricksDialog, getDatabricksValidationSchema } from "../components/integrations/dialogs/databricksDialog";
+import { ECRDialog, getECRValidationSchema } from "../components/integrations/dialogs/ecrDialog";
+import { EmailDialog, getEmailValidationSchema } from "../components/integrations/dialogs/emailDialog";
+import GCPDialog, { getGCPValidationSchema } from "../components/integrations/dialogs/gcpDialog";
+import { GCSDialog, getGCSValidationSchema } from "../components/integrations/dialogs/gcsDialog";
+import { LambdaDialog, getLambdaValidationSchema } from "../components/integrations/dialogs/lambdaDialog";
+import { MariaDbDialog, getMariaDBValidationSchema } from "../components/integrations/dialogs/mariadbDialog";
+import { MongoDBDialog, getMongoDBValidationSchema } from "../components/integrations/dialogs/mongoDbDialog";
+import { MysqlDialog, getMySQLValidationSchema } from "../components/integrations/dialogs/mysqlDialog";
+import OnDemandKubernetesDialog, { getOnDemandKubernetesValidationSchema } from "../components/integrations/dialogs/onDemandKubernetesDialog";
+import { PostgresDialog, getPostgresValidationSchema } from "../components/integrations/dialogs/postgresDialog";
+import { RedshiftDialog, getRedshiftValidationSchema } from "../components/integrations/dialogs/redshiftDialog";
+import { S3Dialog, getS3ValidationSchema } from "../components/integrations/dialogs/s3Dialog";
+import { SlackDialog, getSlackValidationSchema } from "../components/integrations/dialogs/slackDialog";
+import { SnowflakeDialog, getSnowflakeValidationSchema } from "../components/integrations/dialogs/snowflakeDialog";
+import { SparkDialog, getSparkValidationSchema } from "../components/integrations/dialogs/sparkDialog";
+import { getSQLiteValidationSchema, SQLiteDialog } from "../components/integrations/dialogs/sqliteDialog";
+import { AqueductDocsLink } from "./docs";
+import { ServiceInfoMap, ServiceLogos, IntegrationCategories } from "./integrations";
+
+const addingIntegrationLink = `${AqueductDocsLink}/integrations/adding-an-integration`;
+
+export const SupportedIntegrations: ServiceInfoMap = {
```

Comment:
Moved this out into it's own file since we're running into issues with the integrations.ts vs integrations.tsx.